### PR TITLE
Updating to rubocop 0.37

### PR DESCRIPTION
**Pull request by fh**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+  Exclude:
+    - 'easybib/libraries/config.rb'
 
 # Offense count: 9
 # Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: format, sprintf, percent
 Style/FormatString:
-  Enabled: false
+  Exclude:
+    - 'apt/providers/repository.rb'
```
Comment (till) [inline .rubocop_todo.yml]: I think we have a general exclude this that this should be part off.

File: apt/recipes/cacher-client.rb [hunk 2]
```diff
@@ -43,7 +43,7 @@ class ::Chef::Recipe
   end
 end
 
-unless Chef::Config[:solo] || servers.length > 0
+unless Chef::Config[:solo] || !servers.empty?
```
Comment (till) [inline apt/recipes/cacher-client.rb]: Is this our recipe?

File: stack-qa/recipes/deploy-sinopia-nginx.rb [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
   next unless allow_deploy(application, 'sinopia', 'npm-proxy')
 
-  if node.fetch('sinopia', {})['listen'].nil?
-    sinopia_listener = '127.0.0.1:4873'
-  else
-    sinopia_listener = node['sinopia']['listen']
-  end
+  sinopia_listener = if node.fetch('sinopia', {})['listen'].nil?
```
Comment (till) [inline stack-qa/recipes/deploy-sinopia-nginx.rb]: Is this "better" ruby code now?

File: apt/recipes/cacher-client.rb (hunk 2 above)
Comment (fh) [inline apt/recipes/cacher-client.rb]: Partially, see /VERSIONS.md

File: stack-qa/recipes/deploy-sinopia-nginx.rb (hunk 3 above)
Comment (fh) [inline stack-qa/recipes/deploy-sinopia-nginx.rb]: Different. I personally consider it to be nicer as in better readable, but not sure. See RfC in the Ops flow

**Pull request merged**